frontend/src/lib/env.js

```diff
@@ -7,11 +7,12 @@ if (typeof process.env.NODE_ENV === 'undefined') {
 }
 
 const env = {
-  NODE_ENV: process.env.NODE_ENV,
-  API_ROOT: '/api/'
+  NODE_ENV: process.env.NODE_ENV
 };
 
-if (env.NODE_ENV === 'circleci' || env.NODE_ENV === 'development') {
+if (process.env.hasOwnProperty('APP_URL')) {
+  env.API_ROOT = `${process.env.APP_URL}/api`;
+} else if (env.NODE_ENV === 'circleci' || env.NODE_ENV === 'development') {
   env.API_ROOT = 'http://localhost:3000/api/';
```

Comment:
I take that back. We are assuming that APP_URL is always defined for other environments, which is fine.